pkgs/tools/text/vale/default.nix

```diff
@@ -26,6 +34,16 @@ buildGoModule rec {
   # Tests require network access
   doCheck = false;
 
+  passthru.withStyles = selector: symlinkJoin {
```

Comment:
In the `postInstall` of vale, we are currently copying the styles from `/testdata` into a separate `data` output. The [README of that dir](https://github.com/errata-ai/vale/blob/v3/testdata/styles/README.md) says the styles are for testing only and should not be used. Now that we have the real styles packaged, should we just remove the `data` output?

Reply:
Good catch! Yes, it makes sense, I've removed the data output.